plugins/inputs/opcua/opcua_client.go

```diff
@@ -48,13 +49,24 @@ type OpcUA struct {
 }
 
 // OPCTag type
-type OPCTag struct {
-	Name           string `toml:"name"`
+type NodeSettings struct {
+	FieldName      string `toml:"field_name"`
```

Comment:
backwards support for name might be good here.

Reply:
done